### ISSUE TITLE
SAN certificates

### ISSUE THREAD

**Issue by zenhack** [zenhack/simp_le#24]
<a href="https://github.com/ManuelGysin"><img src="https://avatars.githubusercontent.com/u/2873373?v=3" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [ManuelGysin](https://github.com/ManuelGysin)**
_Saturday Jun 04, 2016 at 23:29 GMT_
_Originally opened as https://github.com/kuba/simp_le/issues/109_

----

Hello

What exactly the parameters to create a SAN certificate?

Using multiple -d hosts does not work and tries to create for each host a certificate.

Thanks
Manuel

**Comment by zenhack** [comment]
<a href="https://github.com/svenk"><img src="https://avatars.githubusercontent.com/u/905848?v=3" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [svenk](https://github.com/svenk)**
_Friday Dec 02, 2016 at 11:05 GMT_

----

For me exactly `-d` works, as it is described in https://github.com/kuba/simp_le/blob/master/simp_le.py#L841 and the example in https://github.com/kuba/simp_le/blob/master/simp_le.py#L1435, that is, I can call
```
$ simple_le --default_root /path/to/my/webroot/ -d example.com -d www.example.com
```
and then get
```
$ openssl x509 -in my/certs/for/example.com/cert.pem -text
...
            X509v3 Subject Alternative Name: 
                DNS:example.com, DNS:www.example.com
```
Which is exactly what I think Subject Altnames should do.

In my configuration, those virtual hosts share the same `.well-known/acme-challenge` directory.

**Comment by zenhack** [comment]
Closing this, since @ManuelGysin closed the corresponding issue kuba/simp_le#109
